src/util.js

```diff
@@ -5,5 +5,5 @@ export function parseDateTime(datetime) {
   let [hour, minute, second] = time.split(":");
   let [day, month, year] = date.split("-");
 
-  return new Date(year, parseInt(month)-1, day, hour, minute, second);
+  return new Date(Date.UTC(year, parseInt(month)-1, day, hour, minute, second));
```

Comment:
The dates that are stored in Home Assistant are already in local time. There is no conversion (but there should be) to UTC dates ([source](https://github.com/balloob/home-assistant/blob/master/homeassistant/components/recorder.py#L208)). Where are you seeing this problem?